src/utils/certify.js

```diff
@@ -44,9 +44,14 @@ export default {
         certifiedTokens[tokenId].unity = unity
     },
     getTokenId(name) {
+        let upperName = name.toUpperCase().trim()
+        let index = upperName.indexOf('TOKEN')
+        if (index !==-1 && upperName[index-1] === ' ' && index + 5 === upperName.length) {
+            upperName = upperName.slice(0,index).trim()
+        }
```

Comment:
If a token called `tokenny token`, it will failed to get token ID.
These code can be modify as the following:

```
let len = upperName.length
let endName = len > 6 ? upperName.substring(len - 6, len) : ''
if (endName === ' TOKEN') {
    upperName = upperName.substring(0, len - 6).trim()
}
```